git-extra/git-prompt.sh

```diff
@@ -1,3 +1,5 @@
+. "$HOME"/bash_completion.d/*-completion.bash
```

Comment:
Does this even work when there are more than one scripts in that directory, or when that directory does not even exist?

I think we need this instead:

```sh
for c in "$HOME"/bash_completion.d/*.bash
do
    # Handle absence of any scripts (or the folder) gracefully
    test ! -f "$c" ||
    . "$c"
done
```

Reply:
> NIH syndrome?

No, just usually disagreeing with your coding taste.

Reply:
We're saying the same thing.